Sources/Views/ALKChatCell.swift

```diff
@@ -378,6 +395,13 @@ public final class ALKChatCell: SwipeTableViewCell, Localizable {
         messageLabel.leadingAnchor.constraint(equalTo: emailIcon.trailingAnchor, constant: 0).isActive = true
         messageLabel.trailingAnchor.constraint(equalTo: muteIcon.leadingAnchor, constant: -8).isActive = true
 
+        // setup constrain of custom tags view
+        customTagView.translatesAutoresizingMaskIntoConstraints = false
+        customTagView.topAnchor.constraint(equalTo: messageLabel.bottomAnchor, constant: 10).isActive = true
+        customTagView.leadingAnchor.constraint(equalTo: messageLabel.leadingAnchor, constant: 0).isActive = true
+        customTagView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -8).isActive = true
+        customTagView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor,constant: -8).isActive = true
```

Comment:
Create Padding type for storing padding values.

Reply:
done